### ISSUE TITLE
Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup

### ISSUE THREAD

**Issue by marceloamp2** [ivanvermeyen/laravel-google-drive-demo#17]
Hi,
Following all the steps of your code more when I try to list has this message:

{
"error": {
"errors": [
{
"domain": "usageLimits",
"reason": "dailyLimitExceededUnreg",
"message": "Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup.",
"extendedHelp": "https://code.google.com/apis/console"
}
],
"code": 403,
"message": "Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup."
}
}

**Comment by ivanvermeyen** [comment]
This seems to be related to your API keys/tokens, since it says "unauthenticated".
Can you double check the env/config values?

**Comment by marceloamp2** [comment]
My env:

FILESYSTEM_CLOUD=google
GOOGLE_DRIVE_CLIENT_ID=xxxxxxxxxxxxxx.apps.googleusercontent.com
GOOGLE_DRIVE_CLIENT_SECRET=xxxxxxxxxxxxxxxxxx
GOOGLE_DRIVE_REFRESH_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
GOOGLE_DRIVE_FOLDER_ID=null

**Comment by ivanvermeyen** [comment]
Did you try with this demo project?
Did you get access before the error occurred? (did it work at some point?)

It could be because the keys are invalid, or because of a service not being enabled.
See https://github.com/nao-pon/flysystem-google-drive/issues/22

It could also be an issue with the selected scopes when you created the keys.
Make sure only the first one is selected.

![](https://user-images.githubusercontent.com/3598622/28462312-fa4397ea-6e1a-11e7-93ad-365b891052a6.png)

**Comment by marceloamp2** [comment]
Did you try with this demo project?

**Yes**

Did you get access before the error occurred? (did it work at some point?)

**Never worked**

It could be because the keys are invalid, or because of a service not being enabled.
See nao-pon/flysystem-google-drive#22
(note that Google+ API is not enabled in my account, so I don't think that is necessary)

**I activated**

It could also be an issue with the selected scopes when you created the keys.
Make sure only the first one is selected.

**I did exactly that way you passed**

**Comment by ivanvermeyen** [comment]
I just created a new project in my Google console, added new OAuth2 credentials and generated a refresh token with the playground and I still get a file listing in the project.

So the problem must be with the account.

I did notice that when I double clicked the refresh token in the input field, it did not select the entire key. The first part "1/" was not selected.

I wish I could be of better help than telling you to "check your configuration" but I didn't really find any other possible causes online. Maybe try and create some fresh credentials from scratch?
